consensus-types/blocks/execution.go

```diff
@@ -378,6 +398,16 @@ func (executionPayloadHeader) ValueInGwei() (uint64, error) {
 	return 0, consensus_types.ErrUnsupportedField
 }
 
+// DepositReceipts --
+func (e executionPayloadHeader) DepositReceipts() ([]*enginev1.DepositReceipt, error) {
+	return nil, consensus_types.ErrUnsupportedField
+}
```

Comment:
The header won't have the receipts, it just has a `deposit_receipts_root: Root`. Same goes for `withdrawal_requests_root` below.

Reply:
yep, that's why it returns ErrUnsupportedField. All of these have to comply with the same interface